sass/_QuerySuggestPreview.scss

```diff
@@ -81,7 +84,7 @@
       padding: 10px;
       font-weight: normal;
       margin-top: 0;
-      text-align: center;
+      padding-left: 12px;
```

Comment:
Is there a variable for the margin/padding around each preview that we could reference here?

Reply:
The variable is defined in `MagicBox.scss` with the name `height` as a function parameter. Should I extract it?

Reply:
Hmm, that does not seem like the right variable to reference. Wouldn't `$margin` inside the .coveo-preview-layout make more sense?

In _QuerySuggestPreview.scss on lines 45-46 I see:
```
.coveo-preview-layout {
    $margin: 10px;
    ...
}
```

Reply:
For some reason, I remembered as if you wanted me to make this padding the same as the padding in suggestions. Using `$margin` would indeed make it match up with previews. I'll do that.